### PR TITLE
embeddingsRoutes fix, usage counts added to dataToEmbeddingResponse

### PR DESCRIPTION
**Pull request by cryptocake**
- index.js changed app.use('/v1/embeddings', embeddingsRoutes); to app.use(/^\/v1(?:\/.+)?\/embeddings$/, embeddingsRoutes); to process embeddings like: https://api.openai.com/v1/engines/text-embedding-ada-002/embeddings

- utils.js added usage - prompt_tokens / completion_tokens / total_tokens to the dataToEmbeddingResponse.

- embeddingsRoutes.js added promptTokens and completionTokens count logic.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -146,7 +146,7 @@ app.use(
 );
 app.use('/v1/models', modelsRoutes);
 app.use('/v1/chat', chatRoutes);
-app.use('/v1/embeddings', embeddingsRoutes);
+app.use(/^\/v1(?:\/.+)?\/embeddings$/, embeddingsRoutes);
```
Comment (keldenl) [inline index.js:149]: what's the purpose of this change?

File: routes/embeddingsRoutes.js [hunk 2]
```diff
@@ -76,6 +76,8 @@ router.post('/', async (req, res) => {
 		? req.body.input.join(' ')
 		: req.body.input;
 	const scriptArgs = ['-m', modelPath, '-p', input.replace(/"/g, '\\"')];
+	const promptTokens = Math.ceil(input.length / 4);
+	let completionTokens = 0;
```
Comment (keldenl) [inline routes/embeddingsRoutes.js:80]: looking at the official api: https://platform.openai.com/docs/api-reference/embeddings/create#embeddings/create-model

doesn't appear that we need completion tokens here (also doesn't quite make sense), can you please remove completion token?

File: utils.js [hunk 3]
```diff
@@ -60,7 +60,11 @@ export const dataToResponse = (
 	};
 };
 
-export const dataToEmbeddingResponse = (output) => {
+export const dataToEmbeddingResponse = (
```
Comment (keldenl) [inline utils.js:63]: same comment h ere about completion token, total_tokens would just be prompt_tokens i guess

File: routes/embeddingsRoutes.js (hunk 2 above)
Comment (cryptocake) [reply routes/embeddingsRoutes.js]: I've emoved it and updated.

File: utils.js (hunk 3 above)
Comment (cryptocake) [reply utils.js:63]: Removed and updated.

File: index.js (hunk 1 above)
Comment (cryptocake) [reply index.js:149]: It is in my PR comment:

index.js changed app.use('/v1/embeddings', embeddingsRoutes); to app.use(/^/v1(?:/.+)?/embeddings$/, embeddingsRoutes); to process embeddings like: https://api.openai.com/v1/engines/text-embedding-ada-002/embeddings

Auto-gpt also uses those API calls.

**Review by keldenl — APPROVED** [review]
LGTM
